painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -28,6 +29,9 @@ class PaintingActivity : AppCompatActivity() {
 
         viewModel.painting.observe(this, DataObserver<Painting> { painting ->
             title = painting.title
+            titleArtistTextView.text = "${painting.title} // ${painting.artist.name}" // TODO: string format, but also PaintingUiModel
```

Comment:
rather than doing any logic in this View (the activity), the ViewModel should emit `PaintingUiModel` or something with the fallback logic taken care of.

I'm not sure whether the String formatting belongs in the View or the ViewModel though 🤔